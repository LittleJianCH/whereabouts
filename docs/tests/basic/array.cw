query (abc) {
  equation abc = ["a", "b", "c"]
}

query (a, b, c) {
  equation [a, b, c] = ["a", "b", "c"]
}

query (empty) {
  equation empty = []
}
