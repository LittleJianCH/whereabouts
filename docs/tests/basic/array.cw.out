{ "solutions": [{ "abc": ["a", "b", "c"] }] }
{ "solutions": [{ "a": "a", "b": "b", "c": "c" }] }
{ "solutions": [{ "empty": [] }] }
